v2/mongodb-to-googlecloud/src/test/java/com/google/cloud/teleport/v2/mongodb/templates/MongoDbToBigQueryIT.java

```diff
@@ -152,6 +153,11 @@ public void testMongoDbToBigQuery() throws IOException {
     List<Document> mongoDocuments = generateDocuments();
     mongoDbClient.insertDocuments(collectionName, mongoDocuments);
 
+    String udfFileName = "transform.js";
+    artifactClient.uploadArtifact(
+        "input/" + udfFileName,
+        Resources.getResource("MongoDbToBigQuery/" + udfFileName).getPath());
```

Comment:
This looks good.

A nit but it might be preference.
I recently changed a few tests to use:

```suggestion
    String udfFileName = "transform.js";
    artifactClient.createArtifact(
        "input/" + udfFileName,
        "function transform(inJson) {\n"
            + "    var outJson = JSON.parse(inJson);\n"
            + "    outJson.udf = \"out\";\n"
            + "    return JSON.stringify(outJson);\n"
            + "}");
```


IMHO test is more readable / self-contained, so easier to digest.

Reply:
I like that better. Fixed.